src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -1,60 +1,88 @@
-  preparerId: '231',
-  odometerReading: 123,
-  vehicleConfiguration: 'none',
-  testStationType: 'none',
-  reasonForCancellation: 'none',
-  testerName: '213',
+const testResultModelExample = {
   vrm: '123',
   testStationPNumber: '123',
-  numberOfSeats: 123,
   testerEmailAddress: '123@123.com',
-  euVehicleCategory: '123',
   vin: '213',
-  vehicleSize: '213',
-  preparerName: '213'
-}
+  vehicleSize: '213'
+} as TestResultModel;
 
 describe('GetVehicleTestResultModel', () => {
-  test('action should have the right type and payload when used', () => {
-    const actionInstance = new GetVehicleTestResultModel(testResultModelExample);
+  it('action should have the right type and payload when used', () => {
+    const actionInstance = new GetVehicleTestResultModel('23434234');
 
     expect(actionInstance.type).toBe(EVehicleTestResultModelActions.GetVehicleTestResultModel);
-    expect(actionInstance.payload).toBe(testResultModelExample)
+    expect(actionInstance.payload).toBe('23434234');
   });
 });
 
 describe('GetVehicleTestResultModelSuccess', () => {
-  test('action should have the right type and payload when used', () => {
-    const actionInstance = new GetVehicleTestResultModelSuccess(testResultModelExample);
+  it('action should have the right type and payload when used', () => {
+    const actionInstance = new GetVehicleTestResultModelSuccess([testResultModelExample]);
 
-    expect(actionInstance.type).toBe(EVehicleTestResultModelActions.GetVehicleTestResultModelSuccess);
-    expect(actionInstance.payload).toBe(testResultModelExample)
+    expect(actionInstance.type).toBe(
+      EVehicleTestResultModelActions.GetVehicleTestResultModelSuccess
+    );
+    expect(actionInstance.payload).toStrictEqual([testResultModelExample]);
   });
 });
 
 describe('GetVehicleTestResultModelFailure', () => {
-  test('action should have the right type and payload when used', () => {
-    const actionInstance = new GetVehicleTestResultModelFailure(testResultModelExample);
+  it('action should have the right type and payload when used', () => {
+    const actionInstance = new GetVehicleTestResultModelFailure('test');
 
-    expect(actionInstance.type).toBe(EVehicleTestResultModelActions.GetVehicleTestResultModelFailure);
-    expect(actionInstance.payload).toBe(testResultModelExample)
+    expect(actionInstance.type).toBe(
+      EVehicleTestResultModelActions.GetVehicleTestResultModelFailure
+    );
+    expect(actionInstance.payload).toBe('test');
   });
-});
\ No newline at end of file
+});
+
+describe('SetCurrentState', () => {
+  it('action should have correct type & payload', () => {
+    const actionInstance = new SetCurrentState(VIEW_STATE.VIEW_ONLY);
+
+    expect(actionInstance.type).toBe(EVehicleTestResultModelActions.SetCurrentState);
+    expect(actionInstance.editState).toBe(VIEW_STATE.VIEW_ONLY);
+  });
+});
+
+describe('UpdateTestResult', () => {
+  it('action should have correct type & payload', () => {
```

Comment:
same comment for the unit test description, it would be nice to be a bit more meaningful and explicit on what it is doing I think.